### PR TITLE
prometheus: update to 2.3.1

### PR DESCRIPTION
**Pull request by iaroki**

### CODE REVIEW

**Comment by cr6git** [comment]
2.3.1 is out!

**Comment by iaroki** [comment]
Updated

File: srcpkgs/prometheus/template [hunk 1]
```diff
@@ -15,7 +15,7 @@ license="Apache-2.0"
 homepage="https://prometheus.io/"
```
Comment (maxice8) [inline srcpkgs/prometheus/template]: move homepage= to above distfiles=

Comment (iaroki) [reply srcpkgs/prometheus/template]: Done

**Pull request merged**